introduction.md

```diff
@@ -2,3 +2,11 @@ Hello, my name is Ben, a recent college graduate who would like to master Github
 
 My favorite Code School path (other than Git) is HTML/CSS, where I'm learning Bootstrap.
 
+## Deadly Skills
+
+* HTML
+* CSS
+* XML
+* Linux commands
+
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?